.github/workflows/Build.yml

```
@@ -12,10 +12,10 @@ jobs:
     steps:
       - name: Get the code
         uses: actions/checkout@v2
-      - name: Install NodeJS
-        uses: actions/setup-node@v1
+      - name: Setup NodeJS
+        uses: actions/setup-node@v2
         with:
-          node-version: '14.15.5'
+          node-version: '16'
```

the whole cache section can be replaced with this:
```suggestion
          node-version: '16'
          cache: 'npm'
```

---

@eddiemonge  `cache: 'npm'` doesn't work when no package-lock.json is checked in at the project root.